FRCore/FRCore/Network/FRSecurityConfiguration.swift

```diff
@@ -104,6 +104,22 @@ public class FRSecurityConfiguration: NSObject {
         return false
     }
     
+    /// Extracts public key from the certificate
+    private func publicKey(for certificate: SecCertificate) -> SecKey? {
+        if #available(iOS 12.0, *) {
+            return SecCertificateCopyKey(certificate)
+        } else if #available(iOS 10.3, *) {
+            return SecCertificateCopyPublicKey(certificate)
+        } else {
+            var possibleTrust: SecTrust?
```

Comment:
I am guessing we need to add that cause on SPM we cannot define minor versions. What happens if SPM is configured for 10.0, but the frameworks specify 10.3? Will it allow to get the Package and the project not run? Or would you get an error when checking out? The reason I am asking is cause, it might be better to go to 11 all-together or force 10.3 to avoid code we won't be able to test.

To my next point do we have a way to test the 10.0?

Reply:
@spetrov what do you think of this regarding iOS 10.0?

Reply:
@george-bafaloukas-forgerock when initially I imported SPM (configured .v10) with frameworks that had iOS version set 10.3, it didn't build and and gave this error "'SecCertificateCopyPublicKey' is only available in iOS 10.3 or newer" and suggested to check for #availability. 

Seems like SPM doesn't look at framework versions, only what's set in the swift.package file. I tried this by setting my test project deployment target to 10.0, while the frameworks versions were set 10.3 and it built and run perfectly. So it only cared about the version set in SPM.

I also checked iOS 10 install base statistics, it's less than 0.5% . I am for moving to iOS 11.0 everywhere.

Reply:
OK let's discuss this with @lsanches, @spetrov, Giessel and Bela. I am for going iOS 11 and save the hassle